src/docs/sdk/performance/index.mdx

```diff
@@ -18,7 +18,11 @@ Reference implementations:
 
 This section describes the options SDKs should expose to configure tracing and performance monitoring.
 
-Tracing is enabled by setting either one of two new SDK config options, `tracesSampleRate` and `tracesSampler`. If not set, both default to `undefined`, making tracing opt-in.
+Tracing is enabled by setting any of three SDK config options, `enablePerformance`, `tracesSampleRate` and `tracesSampler`. If not set, both default to `undefined`, making tracing opt-in.
```

Comment:
```suggestion
Tracing is enabled by setting any of three SDK config options, `enablePerformance`, `tracesSampleRate` and `tracesSampler`. If not set, these options default to `undefined`, making tracing opt-in.
```

Reply:
Also wanted to mention that not all languages support `undefined` as a fallback (like Go).